package.json

```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
+    "assign": "git://github.com/newoga/assign.git",
```

Comment:
If we continue to pursue this PR, we should probably tag release rather then work of master at the very least and possibly publish to npm too. Though just wanted to run this implementation by you first (and confirm what we should name these).